storybookjs/storybook
# Build: Add NX bootstrap optimization

## What I did

This PR adds Nx to run the prepare stage. Nx has remote caching which would allow developers to reuse builds done on the CI server. This will make getting started with the repo as well as CI faster.

For more information about Nx, see the [official documentation](https://nx.dev/)

## How to test

Running `nx run-many --target prepare --parallel --all` should successfully build all projects.

Closes https://github.com/storybookjs/storybook/pull/14504
